lib/table_display.rb

```diff
@@ -105,15 +105,6 @@ def to_table_display(*args)
 
 module Kernel
   def pt(target, *options)
-    puts target.respond_to?(:to_table_display) ? target.to_table_display(*options) : target
+    puts target.class.ancestors.include?(Enumerable) ? TableDisplay.to_table_display(target, *options) : target
```

Comment:
I think this would be normally implemented using `target.is_a?(Enumerable)` or perhaps just `target.respond_to?(:each)`.

Reply:
Thanks, good point -- though `to_table_display` calls both `each` and `collect` on the target, so I've refactored the suitability test into its own method.